redisplus/graph/commands.py

```diff
@@ -61,3 +61,6 @@ def config(self, name, value=None, set=False):
             else:
                 raise DataError("``value`` can be provided only when ``set`` is True")
         return self.execute_command("GRAPH.CONFIG", *params)
+
+    def list(self):
```

Comment:
Gah... This is one of those things I'm torn about. list is a base type, so IMHO we shouldn't support a function with the same name as a base type. How about renaming the function **ls**. I realize this isn't ideal, but at least it's unix-like.

Reply:
@chayim What do you say about `listGraphKeys`?

Reply:
I could accept list_keys. That's sane.